### PR TITLE
User can see saved times

### PR DESCRIPTION
**Pull request by LauraRe**
#### Description
_As a User
In order to keep track of my progress
I would like to see my saved times_

Changes proposed in this pull request:

* get saved timesheets from api
* use Stockholm timezone

#### What I have learned working on this feature: 
* how to mock api response in cypress
* how you use the spread operator to add item as first object in an array
* how to set a timezone with moment timezone package

#### Screenshots:
image

### CODE REVIEW

**Comment by amandagibson** [comment]
Why isn't the customer number populating though?

File: src/components/timeTrackingTable.js [hunk 1]
```diff
@@ -30,13 +83,20 @@ export class TimeTrackingTable extends Component {
 			project: this.state.project,
 			activity: this.state.activity,
 			description: "description",
-			fixedRate: "0.0",
+			fixedRate: "",
```
Comment (vick3d) [inline src/components/timeTrackingTable.js]: this change is messing up our unit tests and the build on semaphore.
So either switch back or fix the tests.

Comment (LauraRe) [reply src/components/timeTrackingTable.js]: fixed.

**Comment by LauraRe** [comment]
> Why isn't the customer number populating though?

Because the timesheets api get response does not include it.

**Pull request merged**